Assets/General/Scripts/Managers/UniversalManagers/MainUniversalManagers/TestingScript/AudioManagerTester.cs

```diff
@@ -0,0 +1,28 @@
+using FMOD.Studio;
+using FMODUnity;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AudioManagerTester : MonoBehaviour
```

Comment:
Seeing as this is a testing class I don't have comments, just don't forget to get rid of it when you are done with it after reviews